### DISCUSSION TITLE
ReadsInPromoter larger than nFrags

### DISCUSSION THREAD

**Discussion by pchiang5** [GreenleafLab/ArchR#1705] [General]
Hello,

I got the following output from my ArchR project:

`adata.obs.ReadsInTSS.mean()`

> 4759.797468354431

`adata.obs.nFrags.mean()`

> 20011.732067510547

`adata.obs.PromoterRatio.mean()`

> 1.043950311252914

`adata.obs.ReadsInPromoter.mean()`

> 34353.31012658228

My understanding is that the ReadInPromoter shall always smaller than the total fragment numbers (nFrags), as ReadsInTSS be. However, the mean of  ReadInPromoter was larger than nFrags in this particular dataset and PromoterRatio of some cells are even 2.35 . Is there any cause for this? Thank you.

**Comment by rcorces** [comment]
Mean probably isnt the right metric to be using here. For any given cell, the number of ReadsInPromoter cannot be greater than `2 * nFrags`

Also, fragments are different than reads. 1 fragment = 2 reads.
